src/main/java/org/folio/linked/data/validation/dto/LccnValidator.java

```diff
@@ -0,0 +1,17 @@
+package org.folio.linked.data.validation.dto;
+
+import static org.apache.commons.collections4.CollectionUtils.isEmpty;
+
+import jakarta.validation.ConstraintValidator;
+import java.lang.annotation.Annotation;
+import org.folio.linked.data.domain.dto.LccnRequest;
+
+public abstract class LccnValidator<A extends Annotation, T> implements ConstraintValidator<A, T> {
+
+  protected boolean isCurrent(LccnRequest lccnRequest) {
```

Comment:
[Let's not use inheritance if it's possible to use composition.](https://en.wikipedia.org/wiki/Composition_over_inheritance)

Here we could just extract a method as a static one to Validation util or smth

Reply:
moved to LccnUtils